drivers/volume/portworx/portworx.go

```diff
@@ -1116,6 +1117,73 @@ func (d *portworx) GetClusterPairingInfo() (map[string]string, error) {
 	return pairInfo, nil
 }
 
+func (d *portworx) DecommissionNode(n node.Node) error {
```

Comment:
Did you test this on a jenkins job?

Reply:
running https://jenkins.portworx.co/job/Torpedo/view/Torpedo%20Dashboard/job/tp-2-1-basic-vol-driver/50/console

Reply:
@thiguetta looks like ^^ job failed on the new decommission test. Can you chase this one down?